sim/midas/src/main/cc/emul/mmio.cc

```diff
@@ -86,25 +82,3 @@ bool mmio_t::write_resp() {
     return true;
   }
 }
-
-extern uint64_t main_time;
```

Comment:
This has been moved into simif_emul_t, which has a more functionally consistent scope.